lib/src/js/process_elements.sh

```diff
@@ -1,3 +1,8 @@
 #!/usr/bin/env bash
 
-node packages/custom_element_apigen/src/js/analyze.js $*
+X=$PWD
```

Comment:
that's a good idea indeed. probably this will get the thing work on windows too ?